doc/articles/Guides/CustomPlayer.md

```diff
@@ -0,0 +1,71 @@
+# Customizable Players
+
+In many games you want to customize character creation.  You may want to pick the color of the hair, eyes, skin, height, race, etc.
+
+Out of the box Mirror will instantiate the player for you, which might prevent you from customizing your character. However Mirror provides the option of overriding player creation to anything.
+
+1) Create a class that extends NetworkManager if you have not done so. For example:
+```cs
+public class MMONetworkManager : NetworkManager
+{
+    ...
+}
+```
+and use it as your Network manager.
+2) Open your Network Manager and disable the "Auto Create Player" boolean.
+3) Create a message that describes your player.  For example:
+```cs
+public class CreateMMOCharacterMessage : MessageBase
+{
+    public Race race;
+    public string name;
+    public Color hairColor;
+    public Color eyeColor
+    ...
+}
+```
+4) Create your player prefabs (as many as you need) and add them to the "Register Spawnable Prefabs" in your Network Manager
```

Comment:
Going along with my suggestion below, instead tell them to create a player prefab and assign it normally to the `playerPrefab field`, still with auto-create unchecked. Give the hint that it would have a component with SyncVar's for the character features above.